### PR TITLE
Tycoons Entry for Final PR

### PR DESCRIPTION
**Pull request by rishiagrawal2609**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

[x] Registration Pull request.

[x] Submission Pull request.

## Checklist ✅  

- [x] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [x] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [x] I have used the Markdown file template to add my information for the Hackathon.
- [x] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [x] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/finalist/Tycoons/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+description: "Hey ! We are Team Tycoons. We are here to compete."
+Team-Quote: "Develop and nurture the ideas through teamwork"
+Application_Type: "Android Application"
+---
+
+Hey ! We are Team Tycoons. We are here to compete. We are team of 5 who worked harder to make this application Happens. We became one of the elite team of our campus by winning 3 prize against 30 + teams.
+
+---
+
+# Project's Name
```
Comment (Uyadav207) [inline content/finalist/Tycoons/index.md]: Project name ?

File: content/finalist/Tycoons/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+#  Rishi Agrawal
+
+_**Role-In-Team**_  : Developer and Version Control
+
+_**Working-Stack**_ : Flutter|Dart|VS Code| Git
+
+_**University/School**_ : SRMIST,Ramapuram
+
+ #  Piyush Mohanta
```
Comment (Uyadav207) [inline content/finalist/Tycoons/index.md]: extra space between # and name

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

**Comment by Uyadav207** [comment]
@rishiagrawal2609 , Do the changes asap

**Comment by Uyadav207** [comment]
DO not change other files

**Comment by rishiagrawal2609** [comment]
I have made the changes as requested

File: content/finalist/Tycoons/index.md (hunk 1 above)
Comment (rishiagrawal2609) [reply content/finalist/Tycoons/index.md]: changed

**Review by Uyadav207 — APPROVED** [review]
welcome

**Pull request merged**